apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -1,39 +1,89 @@
-import { Flex, Label, Tooltip } from "@webstudio-is/design-system";
+import { useState } from "react";
+import type { StyleProperty } from "@webstudio-is/css-data";
+import {
+  Button,
+  Flex,
+  Box,
+  Text,
+  Label,
+  Tooltip,
+  Popover,
+  PopoverContent,
+  PopoverPortal,
+  PopoverTrigger,
+  Separator,
+} from "@webstudio-is/design-system";
+import { UndoIcon } from "@webstudio-is/icons";
 import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
-import { propertyNameColorForSelectedBreakpoint } from "./constants";
-import type { PropertyProps } from "../style-sections";
 
-export const PropertyName = ({ property, label, css }: PropertyProps) => {
+type PropertyProps = {
+  property: StyleProperty | StyleProperty[];
+  label: string;
+  onReset: () => void;
+};
+
+export const PropertyName = ({ property, label, onReset }: PropertyProps) => {
   const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  const [isOpen, setIsOpen] = useState(false);
+  const isPopoverEnabled = isCurrentBreakpoint;
+
+  const labelElement = (
+    <Label
```

Comment:
Would it make sense to create a PropertyLabel component that would have same props as the icon button for colors?

based on the latest update from Taylor (https://share.descript.com/view/POUPnkVZUre) we would be using prop color=preset|inherited|default

Reply:
oh lol I just realized we are inside the PropertyName component and that's exactly what's happening, but its color aware itself, ignore my comment above